### PR TITLE
pen-ultimate sasha gong commit

### PR DESCRIPTION
**Pull request by LenaHenningsen**
> Pull-requests should be ready to merge, that is free of conflicts, and they should pass our testsuite. If you require a specific reviewer you can request a review on the right.

### Please indicate issues related to this PR (if any)
See # .
Close # .

### Checklist
-   The title line explains what these changes do
-   All my changes are marked in the `last_modfied` column
-   The PR has no merge conflicts
-   Travis checks are passing

### CODE REVIEW

File: data/Act.csv [hunk 1]
```diff
@@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
...
 A00932,P0057,AT05,PS00522,,1966..1976,1966..1976,Xie made a joke with the reference of Empty Fort Strategy while doing labor work during the CR.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 A00933,P0263,AT27,PS00523,P0169,1966,1966,Mingxin’s teacher Wang Hanying encouraged him and lent when he said he wanted to try writing critiques and lent him the journals.,SS00207,,2,,,2018-11-04,WH,2018-11-04,WH
-A00934,P0120,AT05,AW00110,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
-A00935,P0120,AT05,AW00111,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
-A00936,P0120,AT05,AW00112,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
+A00934,P0120,AT05,AW00110,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
```
Comment (duncdrum) [inline data/Act.csv]: `,` is not the same as `.` see [the wiki](https://github.com/readchina/ReadingData/wiki/FAQ#we-dont-know-if-mr-x-was-born-in-1961-or-1962-do-i-need-a-new-field-for-that) probably another autoformatting error

File: data/Act.csv [hunk 2]
```diff
@@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
...
+A00935,P0120,AT05,AW00111,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
```
Comment (duncdrum) [inline data/Act.csv]: s.a.

File: data/Act.csv [hunk 3]
```diff
@@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
...
+A00936,P0120,AT05,AW00112,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
```
Comment (duncdrum) [inline data/Act.csv]: s.a.

File: data/Person.csv [hunk 4]
```diff
@@ -1229,3 +1229,4 @@ P0805,Kontarsky,Aloys,de,male,NULL,NULL,NULL,NULL,1931,2017,NULL,NULL,PL0443,NUL
 P0805,康塔尔斯基,,zh,male,NULL,NULL,NULL,NULL,1931,2017,NULL,NULL,PL0443,NULL,NULL,https://de.wikipedia.org/wiki/Aloys_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 P0806,Kontarsky,Alfons,de,male,NULL,NULL,NULL,NULL,1932,2010,NULL,NULL,PL0443,NULL,NULL,https://en.wikipedia.org/wiki/Aloys_and_Alfons_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 P0806,康塔尔斯基,,zh,male,NULL,NULL,NULL,NULL,1932,2010,NULL,NULL,PL0443,NULL,NULL,https://en.wikipedia.org/wiki/Aloys_and_Alfons_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
+P0807,Li,Yizhe,en,male,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,SS00255,170-173,,,"Li Yizhe refers to a group of three authors: Li Zhengtian, ChenYiyang, Wang Xizhe, Guo Hongzhi",,2019-02-18,LH,2019-02-18,LH
```
Comment (duncdrum) [inline data/Person.csv]: can you add their personid here Li Zhengtian P0xxxx, Chen Yigang P0xxxx, etc

**Comment by LenaHenningsen** [comment]
Thanks a lot – so the other question is: how can I enable the text option for colum type? My computer does not want me to enter anything here… because it seems that this will happen again and again… I really made sure not to open this thing in excel or anything yesterday…

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Dienstag, 19. Februar 2019 um 08:51
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Author <author@noreply.github.com>
Betreff: Re: [readchina/ReadingData] pen-ultimate sasha gong commit (#356)

 

@duncdrum requested changes on this pull request.

Your Person.csv file is invalid, it dropped the leading 0 in BCE years. This is likely an auto-formatting problem, see the wiki for how to treat birthyear and deathyear as text. The correct format is -0003, 0123 etc. If the import window look different check and update your version of libbreoffice to the latest 6.2.0.3

In data/Act.csv:
> @@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
 
 A00931,P0056,AT10,PS00521,,195x,195x,Gu was commissioned by Mao Zedong to take charge of the punctuation of the book.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 
 A00932,P0057,AT05,PS00522,,1966..1976,1966..1976,Xie made a joke with the reference of Empty Fort Strategy while doing labor work during the CR.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 
 A00933,P0263,AT27,PS00523,P0169,1966,1966,Mingxin’s teacher Wang Hanying encouraged him and lent when he said he wanted to try writing critiques and lent him the journals.,SS00207,,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00934,P0120,AT05,AW00110,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00935,P0120,AT05,AW00111,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00936,P0120,AT05,AW00112,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
+A00934,P0120,AT05,AW00110,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
 
, is not the same as . see the wiki probably another autoformatting error

In data/Act.csv:
> @@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
 
 A00931,P0056,AT10,PS00521,,195x,195x,Gu was commissioned by Mao Zedong to take charge of the punctuation of the book.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 
 A00932,P0057,AT05,PS00522,,1966..1976,1966..1976,Xie made a joke with the reference of Empty Fort Strategy while doing labor work during the CR.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 
 A00933,P0263,AT27,PS00523,P0169,1966,1966,Mingxin’s teacher Wang Hanying encouraged him and lent when he said he wanted to try writing critiques and lent him the journals.,SS00207,,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00934,P0120,AT05,AW00110,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00935,P0120,AT05,AW00111,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00936,P0120,AT05,AW00112,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
+A00934,P0120,AT05,AW00110,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
 
+A00935,P0120,AT05,AW00111,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
s.a.

In data/Act.csv:
> @@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
 
 A00931,P0056,AT10,PS00521,,195x,195x,Gu was commissioned by Mao Zedong to take charge of the punctuation of the book.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 
 A00932,P0057,AT05,PS00522,,1966..1976,1966..1976,Xie made a joke with the reference of Empty Fort Strategy while doing labor work during the CR.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 
 A00933,P0263,AT27,PS00523,P0169,1966,1966,Mingxin’s teacher Wang Hanying encouraged him and lent when he said he wanted to try writing critiques and lent him the journals.,SS00207,,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00934,P0120,AT05,AW00110,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00935,P0120,AT05,AW00111,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
-A00936,P0120,AT05,AW00112,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
+A00934,P0120,AT05,AW00110,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
 
+A00935,P0120,AT05,AW00111,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
+A00936,P0120,AT05,AW00112,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
 
s.a.

In data/Person.csv:
> @@ -1229,3 +1229,4 @@ P0805,Kontarsky,Aloys,de,male,NULL,NULL,NULL,NULL,1931,2017,NULL,NULL,PL0443,NUL
 
 P0805,康塔尔斯基,,zh,male,NULL,NULL,NULL,NULL,1931,2017,NULL,NULL,PL0443,NULL,NULL,https://de.wikipedia.org/wiki/Aloys_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 
 P0806,Kontarsky,Alfons,de,male,NULL,NULL,NULL,NULL,1932,2010,NULL,NULL,PL0443,NULL,NULL,https://en.wikipedia.org/wiki/Aloys_and_Alfons_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 
 P0806,康塔尔斯基,,zh,male,NULL,NULL,NULL,NULL,1932,2010,NULL,NULL,PL0443,NULL,NULL,https://en.wikipedia.org/wiki/Aloys_and_Alfons_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 
+P0807,Li,Yizhe,en,male,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,SS00255,170-173,,,"Li Yizhe refers to a group of three authors: Li Zhengtian, ChenYiyang, Wang Xizhe, Guo Hongzhi",,2019-02-18,LH,2019-02-18,LH
 
can you add their personid here Li Zhengtian P0xxxx, Chen Yigang P0xxxx, etc

**Comment by LenaHenningsen** [comment]
I think this looks clean now; Li Yizhe will have to wait...

**Pull request merged**